integrations/anthropic/src/haystack_integrations/components/generators/anthropic/chat/chat_generator.py

```diff
@@ -344,16 +345,15 @@ def _convert_chat_completion_to_chat_message(
         )
         return message
 
-    @staticmethod
-    def _convert_anthropic_chunk_to_streaming_chunk(chunk: Any) -> StreamingChunk:
+    def _convert_anthropic_chunk_to_streaming_chunk(self, chunk: Any, component_info: ComponentInfo) -> StreamingChunk:
```

Comment:
1. I would keep this as a `@staticmethod`
2. unrelated: can you try to annotate `chunk` with a more specific type hint and see if typing is still ok (via `hatch run test:types`) or which changes are required...